tests/tasks/verify_settings.yml

```diff
@@ -152,29 +152,15 @@
       changed_when: false
       register: __mssql_conf_get_traceflag
 
-    - name: Print vars to debug weird test failures in the CI
-      debug:
-        var: __mssql_conf_get_traceflag.stdout
-
-    - name: Print vars to debug weird test failures in the CI
-      debug:
-        var: "'3979' in __mssql_conf_get_traceflag.stdout"
-
-    - name: Print vars to debug weird test failures in the CI
-      debug:
-        var: __verify_mssql_is_tuned_for_fua
-
-    - name: Print vars to debug weird test failures in the CI
-      debug:
-        var: >-
-          ('3979' in __mssql_conf_get_traceflag.stdout) ==
-          (__verify_mssql_is_tuned_for_fua)
+    - name: Verify the traceflag setting
+      assert:
+        that: "'3979' in __mssql_conf_get_traceflag.stdout"
+      when: __verify_mssql_is_tuned_for_fua
```

Comment:
`when: __verify_mssql_is_tuned_for_fua | bool` to avoid the bareword problem

or - if `__verify_mssql_is_tuned_for_fua` can be `null` - use 
```yaml
when:
  - __verify_mssql_is_tuned_for_fua is not none
  - __verify_mssql_is_tuned_for_fua | bool
```

Reply:
Will apply `bool` there. This is an internal to role task, so I expect developers who will use this task to set to either `true` or `false`, and not to `none`.